integration_tests/kafka_tests/run.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/bash -e
 
+function section_header() {
+  printf "\n"
+  printf "╓─────[ $@ ]─────╖"
+  printf "\n"
+}
+
 function is_kafka_ready() {
-    if [[ -z "$(docker-compose exec kafka bash -c '/opt/kafka/bin/kafka-topics.sh --zookeeper $KAFKA_ZOOKEEPER_CONNECT --topic rig --describe')" ]]; then
+    if [[ -z "$(docker-compose exec kafka bash -c 'kafka-topics --list --zookeeper zookeeper:2181')" ]]; then
         # no output - Kafka is not ready yet
         return 1
     else
         # Kafka responds!
+        section_header "Creating Kafka topics"
+        docker-compose exec kafka bash -c 'kafka-topics --create --topic rig_test --partitions 1 --replication-factor 1 --if-not-exists --zookeeper zookeeper:2181'
+        docker-compose exec kafka bash -c 'kafka-topics --create --topic rig-proxy-response --partitions 1 --replication-factor 1 --if-not-exists --zookeeper zookeeper:2181'
+        docker-compose exec kafka bash -c 'kafka-topics --create --topic rig_kafka_test_simple_topic --partitions 1 --replication-factor 1 --if-not-exists --zookeeper zookeeper:2181'
+        docker-compose exec kafka bash -c 'kafka-topics --create --topic rigAvro --partitions 1 --replication-factor 1 --if-not-exists --zookeeper zookeeper:2181'
```

Comment:
3 topic names, 3 casings: dashed, snake_case and camelCase ;)

Reply:
Diversity :D